scanpipe/templates/scanpipe/project_form.html

```diff
@@ -32,14 +32,14 @@ <h2 class="subtitle mb-0 pt-2 mb-5">
             <div class="box">
               <label class="label has-text-weight-semibold" for="{{ form.input_files.id_for_label }}">Upload files</label>
               <div class="file has-name is-boxed is-fullwidth">
-                <label class="file-label">
+                <label class="file-label" id="file_upload_label">
```

Comment:
Those template changes needs to be applied to the templates/scanpipe/includes/add_inputs_modal.html file as well.